ironfish/src/rpc/routes/node/getStatus.ts

```diff
@@ -368,6 +376,9 @@ async function getStatus(node: FullNode): Promise<GetNodeStatusResponse> {
         sequence: node.wallet.chainProcessor.sequence ?? -1,
       },
     },
+    wallet: {
+      enabled: node.wallet.isEnabled(),
```

Comment:
Can this just be `config.get('enableWallet')`?

Reply:
ah i thought this class didn't have access to config, i'll try that

Reply:
I don't think file has access to it